### PR TITLE
Adding support for connecting external prometheus server and alert manager

### PR DESCRIPTION
**Pull request by rajdas98**
Adding support for connecting external prometheus server and alert manager
Issue: https://github.com/keptn-contrib/prometheus-service/issues/53
Changes:
 - Refactoring some logic in `eventhandling/configureEvent.go`
 - Rewriting `utils/prometheus.go` after removing all functions and add the following functions:
   - UpdatePrometheusConfigMap()
   - GetConfigMap()
   - DeletePod()
   - CreateConfigMap()
   - CreateAMTempConfigMap()
   - UpdateAMConfigMap()
 - Adding the following environment variables in `deploy/service.yaml`
   -  PROMETHEUS_NS
   - PROMETHEUS_CM
   - PROMETHEUS_LABELS
   - ALERT_MANAGER_CM
   - ALERT_MANAGER_LABELS
   - ALERT_MANAGER_NS

### CODE REVIEW

**Comment by rajdas98** [comment]
cc: @jetzlstorfer @johannes-b @bacherfl

File: deploy/service.yaml [hunk 1]
```diff
@@ -198,7 +191,7 @@ spec:
     spec:
       containers:
         - name: prometheus-service
-          image: keptncontrib/prometheus-service:0.4.0
+          image: imrajdas/prometheus-service:ci
```
Comment (johannes-b) [inline deploy/service.yaml:194]: Please change it to: `keptncontrib/prometheus-service:latest`

File: eventhandling/configureEvent.go [hunk 2]
```diff
@@ -31,13 +30,20 @@ const ErrorRate = "error_rate"
...
 const ResponseTimeP90 = "response_time_p90"
 const ResponseTimeP95 = "response_time_p95"
-const keptnPrometheusSLIConfigMapName = "prometheus-sli-config"
-const podNamespaceEnvName = "POD_NAMESPACE"
 const metricsScrapePathEnvName = "METRICS_SCRAPE_PATH"
 const environmentEnvName = "env"
 
 const sliResourceURI = "prometheus/sli.yaml"
 
+var (
```
Comment (johannes-b) [inline eventhandling/configureEvent.go:38]: While this is technically correct how you are reading the environment variables, can you please change it to the way it is implemented here: https://github.com/keptn/keptn/blob/master/distributor/cmd/main.go#L47

* The benefit is that you can set a default value. 
* Please use CamelCase for variable names.

**Review by johannes-b — COMMENTED** [review]
Please take my comments under consideration. Thanks.

File: deploy/service.yaml (hunk 1 above)
Comment (rajdas98) [reply deploy/service.yaml]: Done

File: eventhandling/configureEvent.go (hunk 2 above)
Comment (rajdas98) [reply eventhandling/configureEvent.go]: Done

**Comment by rajdas98** [comment]
@johannes-b Your requested changes have been pushed. PTAL

**Review by bacherfl — APPROVED** [review]

**Review by johannes-b — APPROVED** [review]
lgtm

**Pull request merged**